.github/workflows/e2e_other_tests.yml

```diff
@@ -0,0 +1,56 @@
+name: End-to-End tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  e2e_other_tests:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.x'
+          architecture: 'x64'
+      - name: Setup python by installing wheel
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: pip install wheel==0.35.0
+      - name: Install third party
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: python -m scripts.install_third_party_libs
+      - name: Install chrome version 88.0.4324.96-1
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: |
+          sudo apt-get update
+          sudo apt-get install libappindicator3-1
+          curl -L -o google-chrome.deb https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_88.0.4324.96-1_amd64.deb
+          sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
+          sudo dpkg -i google-chrome.deb
+          google-chrome --version
+      - name: Run e2e Collections Test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --suite="collections" --prod_env
+      - name: Run e2e Accessibility Test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --skip-install --skip-build --suite="accessibility" --prod_env
+      - name: Run e2e Classroom Page Test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --skip-install --skip-build --suite="classroomPageFileUploadFeatures" --prod_env
+      - name: Run e2e Navigation Test
+        if: startsWith(github.head_ref, 'update-changelog-for-release') == false
+        run: xvfb-run -a python -m scripts.run_e2e_tests --prod_env --deparallelize_terser --skip-install --skip-build --suite="navigation"
```

Comment:
Is the `--deparallelize_terser` still needed? I think it was only for CircleCI

Reply:
removed it!